languages/scala-2/template/build.sbt

```diff
@@ -0,0 +1,16 @@
+name := "algorithm"
+organization := "algorithmia"
+version := "0.1"
+scalaVersion := "2.13.1"
+
+resolvers += "Maven Central" at "http://repo1.maven.org/maven2/org/"
+
+resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"

```

Comment:
http -> https (also line 6)

There is a warning in the compile output about http being deprecated

Reply:
There's also a built in short hand for this repo:
`resolvers += Resolver.typesafeRepo("releases")`

see: https://www.scala-sbt.org/1.x/docs/Resolvers.html